### PR TITLE
llvm@8: upstream Linux fixes

### PR DESCRIPTION
**Pull request by danielnachun**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by danielnachun** [comment]
The fixes for LLVM 9 also fix the build failures in LLVM 8.  The test for Big Sur fails for the same reason as in LLVM 9 as well, so whatever fix we use there should work here.

**Review by carlocab — APPROVED** [review]
Nice work. I suggest we merge this without bottles so that you can get this bottled on Linux, and we can backport any Big Sur fixes from the LLVM 9 PR later on.

**Comment by danielnachun** [comment]
Okay, I will squash and merge.

File: Formula/llvm@8.rb [hunk 1]
```diff
@@ -77,6 +95,7 @@ def install
     (buildpath/"tools/clang/tools/extra").install resource("clang-extra-tools")
     (buildpath/"projects/openmp").install resource("openmp")
     (buildpath/"projects/libcxx").install resource("libcxx")
+    on_linux { (buildpath/"projects/libcxx").install resource("libcxx") }
```
Comment (carlocab) [inline Formula/llvm@8.rb:98]: Oh wait, hang on. Should this be
```ruby
on_linux { (buildpath/"projects/libcxxabi").install resource("libcxxabi") }
```
?

**Pull request merged**